bin/centos71-devbuild.sh

```diff
@@ -44,9 +44,9 @@ if [ "${1}" = "--fg" ]
 then
   echo "CentOS 7.1 Dev Build kicked off, Log being saved in ${LOG_FILE} ..."
 
-  time ruby /build/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
+  time ruby /build/manageiq/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
```

Comment:
Does not matter namewise, if we require the symlink from above, We can also go back to the way it was.